Makefile

```diff
@@ -698,58 +696,43 @@ docker-all: docker-build docker-publish kbld-image-replace ## Ship Docker images
 create-package: ## Stub out new package directories and manifests. Usage: make create-management-package PACKAGE_NAME=foobar
 	@hack/packages/scripts/create-package.sh $(PACKAGE_REPOSITORY) $(PACKAGE_NAME)
 
+.PHONY: prep-package-tools
+prep-package-tools:
+	cd hack/packages/package-tools && $(GO) mod tidy
+
 .PHONY: package-bundle
-package-bundle: ## Build one specific tar bundle package, needs PACKAGE_NAME VERSION
-	PACKAGE_REPOSITORY=$(PACKAGE_REPOSITORY) PACKAGE_NAME=$(PACKAGE_NAME) $(PACKAGES_SCRIPTS_DIR)/package-utils.sh generate_single_imgpkg_lock_output
-	PACKAGE_REPOSITORY=$(PACKAGE_REPOSITORY) PACKAGE_NAME=$(PACKAGE_NAME) PACKAGE_SUB_VERSION=$(PACKAGE_SUB_VERSION) $(PACKAGES_SCRIPTS_DIR)/package-utils.sh create_single_package_bundle
+package-bundle: tools prep-package-tools ## Build one specific tar bundle package, needs PACKAGE_NAME VERSION
+	cd hack/packages/package-tools && $(GO) run main.go package-bundle generate $(PACKAGE_NAME) --repository=$(PACKAGE_REPOSITORY) --version=$(BUILD_VERSION) --sub-version=$(PACKAGE_SUB_VERSION)
```

Comment:
Suggestion: we could just build this binary once in `prep-package-tools`, place it in `bin/` and use it elsewhere in this file. `go run` relinks the binary every time (afaik), so we can gain a small performance improvement by building the binary once and reusing it.

Note that this might need some `Makefile` magic to not build this binary every time like what we do with tooling binaries currently.